legacy/TownyHModFlatFileSource.java

```diff
@@ -1,3 +1,487 @@
+<<<<<<< HEAD:src/com/palmergames/bukkit/towny/db/TownyHModFlatFileSource.java
```

Comment:
You don't need to add anything to the hmodflatfilesource, it is legacy and will probably be moved to the legacy folder soonish